browser/ui/webui/brave_rewards_internals_ui.cc

```diff
@@ -22,7 +22,7 @@
 #include "brave/components/brave_rewards/resources/grit/brave_rewards_internals_generated_map.h"
 #include "brave/components/brave_rewards/resources/grit/brave_rewards_resources.h"
 #else
-#include "components/brave_rewards/settings/resources/grit/brave_rewards_settings_generated_map.h"
+#include "components/brave_rewards/settings/resources/grit/brave_rewards_internals_generated_map.h"
```

Comment:
this can be moved out of if/else as we use the same include

Reply:
Fixed